### PR TITLE
stdlib: Add types for open-uri

### PR DESCRIPTION
**Pull request by tk0miya**

### CODE REVIEW

File: stdlib/open-uri/0/open-uri.rbs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+module URI
+  def self.open: (String name, ?String mode, ?Integer perm, ?untyped options) -> (StringIO | Tempfile)
```
Comment (tk0miya) [inline stdlib/open-uri/0/open-uri.rbs:2]: This return type is a bit incorrect. `URI.open` returns a StringIO (or a Tempfile) object extending by `OpenURI::Meta`. But it's difficult to declare it to me...

File: stdlib/open-uri/0/open-uri.rbs [hunk 2]
```diff
@@ -0,0 +1,545 @@
...
+# Author
+# :   Akira Yamada <akira@ruby-lang.org>
+# Documentation
+# :   Akira Yamada <akira@ruby-lang.org> Dmitry V. Sabanin <sdmitry@lrn.ru>
+#     Vincent Batts <vbatts@hashbangbash.com>
+# License
+# :   Copyright (c) 2001 akira yamada <akira@ruby-lang.org> You can redistribute
+#     it and/or modify it under the same term as Ruby.
+#
+module URI
```
Comment (soutaro) [inline stdlib/open-uri/0/open-uri.rbs:97]:
```suggestion
%a{annotate:rdoc:skip} module URI
```

And run `rbs annotate` (or equivalent) again.

File: stdlib/open-uri/0/open-uri.rbs [hunk 3]
```diff
@@ -0,0 +1,545 @@
...
```
Comment (soutaro) [inline stdlib/open-uri/0/open-uri.rbs:521]: Use `%a{annotate:rdoc:skip}` here too.

File: stdlib/open-uri/0/open-uri.rbs (hunk 2 above)
Comment (tk0miya) [reply stdlib/open-uri/0/open-uri.rbs:2]: Thanks!

File: stdlib/open-uri/0/open-uri.rbs (hunk 1 above)
Comment (tk0miya) [reply stdlib/open-uri/0/open-uri.rbs]: @soutaro taught me the way to represent types for mix-ins, and I updated this definition now.
Resolved.

Thanks as always!

**Pull request merged**